### PR TITLE
update copyright year to 2023

### PR DESCRIPTION
**Pull request by intoeetive**

### CODE REVIEW

File: system/ee/language/english/pro_search_lang.php [hunk 1]
```diff
@@ -1,17 +1,5 @@
 <?php
 
-if (! defined('BASEPATH')) {
```
Comment (TomJaeger) [inline system/ee/language/english/pro_search_lang.php:3]: @intoeetive I'm guessing this was removed on accident?

File: system/ee/language/english/pro_variables_lang.php [hunk 2]
```diff
@@ -1,17 +1,5 @@
 <?php
 
-if (! defined('BASEPATH')) {
```
Comment (TomJaeger) [inline system/ee/language/english/pro_variables_lang.php:3]: @intoeetive I'm guessing this was removed on accident?

**Review by TomJaeger — CHANGES_REQUESTED** [review]
Please see comments @intoeetive

File: system/ee/language/english/pro_search_lang.php (hunk 1 above)
Comment (TomJaeger) [reply system/ee/language/english/pro_search_lang.php:3]: The whole section here.. not just this one line.

File: system/ee/language/english/pro_variables_lang.php (hunk 2 above)
Comment (TomJaeger) [reply system/ee/language/english/pro_variables_lang.php:3]: The whole section here.. not just this one line.

Comment (intoeetive) [reply system/ee/language/english/pro_variables_lang.php:3]: No, it was removed to bring the files to same format that other `lang` files have

**Review by TomJaeger — APPROVED** [review]

**Pull request merged**